### ISSUE TITLE
[BUG]: Screen is flipped vertically and Datachannel dead when close up one of multi-tabs

### ISSUE THREAD

**Issue by lyclyc171** [Unity-Technologies/UnityRenderStreaming#930]
### Package version

3.1.0-exp.6

### Environment

```markdown
* OS: Debian 11
* Unity version:2022.2.2f1c1
* Graphics API: OpenGL
* Browser: FireFox
```

### Steps To Reproduce

1. Running broadcast sample on debian 11
2. Open multiple tabs(>1) on browser
3. Close one of tabs

### Current Behavior

1. Observing screen is flipped vertically
2. Do not receive any mouse and keyboard event

### Expected Behavior

screen and user interface is normal

### Anything else?

Actually I have find the reason by debugging. BroadCast.Disconnect() will be called when close a tab, then following functions RemoveSender and RemoveChannel are called subsequently.
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/90133241/8bdb86b5-c090-4a72-9433-01c6250424eb)
1. RemoveSender lead to call Dispose() in VideoStreamSourceScreen, then m_screenCopyTexture will be destroyed
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/90133241/57ed767a-076d-4f0d-af66-531dbe6413f5)
which is the reference to sourceTexture_ in VideoTrackSource
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/90133241/8dea8cb5-6f57-46b3-9dfb-00d2ec302aba)
so the sourceTexture_  is null and Graphics.Blit will copy a correct screen texture to upside down.

2. As for RemoveChannel, I think there is already a commit to fix it:
[https://github.com/Unity-Technologies/UnityRenderStreaming/pull/925](url)

**Comment by JohnnyNgo** [comment]
@lyclyc171 yeah i had this problem long time ago, and i think it was because of this code in SignalingHandlerBase which was probably removing the track for that one user, then creating a new, wrongfully-flipped track for the other users.
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/43874041/e846cc0c-c8b6-42f3-9f66-6930bc90f98c)
and i also updated the SetTransceiver method in StreamSenderBase too
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/43874041/66d09943-c34a-4dd5-846f-b2a372487060)

**Comment by karasusan** [comment]
@lyclyc171
How about the google chrome?

**Comment by karasusan** [comment]
@JohnnyNgo 
Is this issue already fixed by #925 ?

**Comment by JohnnyNgo** [comment]
@karasusan I'm using Render Streaming 3.1.0-exp.6 for reference, so I don't think I have tried that solution.
I never had the issue of inputs or resolution changes as mentioned in #925, probably because I don't use Broadcast.cs (I made my own custom script to handle removing/adding senders/channels to each users)

So I'd say my solution here is more specific to the Vertical Flipping part and is not already fixed.

**Comment by lyclyc171** [comment]
> @lyclyc171 How about the google chrome?

No, it doesn't work

**Comment by lyclyc171** [comment]
> @lyclyc171 yeah i had this problem long time ago, and i think it was because of this RemoveSender method in SignalingHandlerBase which was probably removing the track for that one user, then recreating a new track which wrongfully flips the texture again for the other users. So I commented it out: ![image](https://user-images.githubusercontent.com/43874041/257326461-e846cc0c-c8b6-42f3-9f66-6930bc90f98c.png) and i also updated the SetTransceiver method in StreamSenderBase too ![image](https://user-images.githubusercontent.com/43874041/257327008-66d09943-c34a-4dd5-846f-b2a372487060.png) Now, this might get things working for you, but I'm not fully sure if I'm fully cleaning up everything memory-wise, so insights here would be appreciated.

Does the screen image turn to normal after you comment out "RemoveTrack"？ Actually my debugging result tell me "OnStroppedStream" ( in your second pic ) is the reason who calls Dispose().

Anyway, thanks for your clue, I will try to comment RemoveTrack out to check result.

**Comment by RaoulBickmann** [comment]
Hey I have a similar issue in 3.1.0-exp.7. When I have multiple tabs open and close one the image for the others is distorted.  
Just changing the SetTransceiver method to how @JohnnyNgo commented 
![](https://user-images.githubusercontent.com/43874041/257327008-66d09943-c34a-4dd5-846f-b2a372487060.png)
seems to fix it for me. (Or just never calling RemoveSender to begin with)
Are there already plans to implement this or sth similar @karasusan and if yes do you know when this might happen?
